inc/tag.class.php

```diff
@@ -70,6 +70,9 @@ public static function getBlacklistItemtype() {
     * @return bool
     */
    public static function canItemtype($itemtype = '') {
+      if (empty($itemtype)) {
+         return false;
+      }
       return (!class_exists($itemtype)
               || !in_array($itemtype, self::getBlacklistItemtype()));
```

Comment:
I think the inial test was wrong.
```suggestion
      return !empty($itemtype) && class_exists($itemtype) && !in_array($itemtype, self::getBlacklistItemtype());
```